src/FormValidation.js

```diff
@@ -0,0 +1,32 @@
+import * as yup from "yup";
```

Comment:
Create a `util` folder next to `src`, and rename this file `validators.js`.

In your react component, you should be able to import it using absolute paths:
```
import { signInValidationSchema } from "util/validators"
```

Reply:
I ended up having to place the util folder inside of the src folder, otherwise, it showed a module exports not found error.